src/screencast/pipewire_screencast.c

```diff
@@ -196,8 +204,17 @@ static const struct pw_stream_events pwr_stream_events = {
 	.param_changed = pwr_handle_stream_param_changed,
 	.add_buffer = pwr_handle_stream_add_buffer,
 	.remove_buffer = pwr_handle_stream_remove_buffer,
+	.process = pwr_handle_stream_process,
 };
 
+void xdpw_pwr_trigger_process(struct xdpw_screencast_instance *cast) {
+	pw_stream_trigger_process(cast->stream);
+}
+
+bool xdpw_pwr_is_driving(struct xdpw_screencast_instance *cast) {
+	return pw_stream_is_driving(cast->stream);
+}
```

Comment:
Do we really need these trivial wrappers?

Reply:
we need a wrapper for `pw_stream_trigger_process`, otherwise we cast a function pointer from `int (*)` to `void (*)`

Reply:
We could make `xdpw_pwr_trigger_process` a noop if we aren't the driver (removing the `xdpw_pwr_is_driving` wrapper), but i'm not happy with waiting for a noop.

Reply:
or we put the whole fps_limit logic into the `.process` callback so we limit the fps even when we are not the driver. But that's sth. for a follow up imho.